packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -848,17 +848,16 @@ Future<T?> showMenu<T>({
   assert(captureInheritedThemes != null);
   assert(debugCheckHasMaterialLocalizations(context));
 
-  String? label;
+  String? label = semanticLabel;
```

Comment:
Why can't we just use `semanticLabel` directly? i.e. remove this line and just use `semanticLabel` instead of `label` below?

Reply:
We can! Changed it.